### PR TITLE
New version: ONLYOFFICE.DesktopEditors version 7.3.0.2

### PR DESCRIPTION
**Pull request by papacarlo**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95920)

### CODE REVIEW

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.0.2
```
Comment (Trenly) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.installer.yaml:5]:
```suggestion
PackageVersion: 7.3.0.184
```

```
--> Comparing ARP Entries

DisplayName                                                        DisplayVersion Publisher             ProductCode
-----------                                                        -------------- ---------             -----------
ONLYOFFICE Desktop Editors 7.3 (x64)                               7.3.0.184      Ascensio System SIA   ONLYOFFICE Desktop Editors_is1
```

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.0.2
```
Comment (Trenly) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.3.0.184
```

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.0.2
```
Comment (Trenly) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.2/ONLYOFFICE.DesktopEditors.yaml:5]:
```suggestion
PackageVersion: 7.3.0.184
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by khewweifeng** [comment]
(1/3) Found ONLYOFFICE Desktop Editors [ONLYOFFICE.DesktopEditors] Version 7.3.0.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.0.1/DesktopEditors_x64.exe
An unexpected error occurred while executing the command:
Download request status is not success.
0x80190194 : unknown error

**Comment by stephengillie** [comment]
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: 20842e15f35a5ec1386201f251f097b5d1c222a0f940289187ffe8270a238fa4`
>   
>   * manifests\o\ONLYOFFICE\DesktopEditors\7.3.0.1
> * `InstallerSha256: 66f92a671c1dfde943de33c0d61de042da78b2508c1d0019eed7ed0b6589db9d`
>   
>   * manifests\o\ONLYOFFICE\DesktopEditors\7.3.0.1

These [files from the previous manifest version](https://github.com/microsoft/winget-pkgs/blob/master/manifests/o/ONLYOFFICE/DesktopEditors/7.3.0.1/ONLYOFFICE.DesktopEditors.installer.yaml) are no longer available for download, and should be removed.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**